easybuild/tools/config.py

```diff
@@ -120,6 +120,7 @@
 DEFAULT_PR_TARGET_ACCOUNT = 'easybuilders'
 DEFAULT_PREFIX = os.path.join(os.path.expanduser('~'), ".local", "easybuild")
 DEFAULT_REPOSITORY = 'FileRepository'
+DEFAULT_EXTRA_SOURCE_URLS = ('https://sources.easybuild.io/',)
```

Comment:
The trailing slash lead to a duplication later (added in the loop using it, wasn't there before) on which might cause issues, see above: `url += "/" + name_letter + "/" + location`

```suggestion
DEFAULT_EXTRA_SOURCE_URLS = ('https://sources.easybuild.io',)
```